portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item (comment) within a list of comments. */
+public final class Comment {
+
+  private final long id;
```

Comment:
from an api point of view it's often better to have a string as your key, people don't make assumption about them, eg that they're sequential, and it allows you to make changes to the way you generate your ids easily by adding a prefix to the front and generating a new set of numbers that you add to the end of the string. 
Eg maybe at the moment you're using the date as a key, but you realise actually you need more numbers than that so now you use the date-time, but to make sure you avoid collisions you put 'dt-' on the front of the id.

Reply:
never mind, just realised this is an id coming from the datastore entity

Reply:
Can I resolve this? Will keep a note of this though for the capstone project :)

Reply:
yeah sorry, resolved, should have looked more closely at where it was coming from before commenting on it.